x-pack/platform/plugins/shared/maps/public/connected_components/mb_map/tooltip_control/features_tooltip/features_tooltip.tsx

```diff
@@ -199,3 +206,16 @@ export class FeaturesTooltip extends Component<Props, State> {
     );
   }
 }
+
+const componentStyles = {
+  actionLinksStyles: ({ euiTheme }: UseEuiTheme) =>
+    css({
+      padding: euiTheme.size.xs,
+    }),
+};
+
+const ActionLinksBox = ({ children }: { children: ReactNode }) => {
+  const styles = useMemoizedStyles(componentStyles);
+
+  return <div css={styles.actionLinksStyles}>{children}</div>;
+};
```

Comment:
Based on our convo offline, I figured I would add a comment summarizing my initial thoughts :)

Definitely defer to @mbondyra's advice here - but I wonder if the overhead of `useMemoizedStyles` is necessary for such a small style? If we are defining `componentStyles` outside of the render scope, I wonder what the benefit of memoizing on `euiTheme` is in this case 🤔 Couldn't we just do this?

```suggestion
const componentStyles = ({ euiTheme }: UseEuiTheme) =>
  css({
    padding: euiTheme.size.xs,
  });

const ActionLinksBox = ({ children }: { children: ReactNode }) => {
  return <div css={componentStyles}>{children}</div>;
};
```

Reply:
I agree (although it'd be good to decide when it's not worth to memoize), and I actually think it would make sense to not extract `ActionLinksBox` but just use the styles directly inside the class component (as a function) 🙏🏼